src/main/java/pl/byczazagroda/trackexpensesappbackend/model/FinancialTransactionCategory.java

```diff
@@ -74,4 +80,11 @@ public FinancialTransactionCategory(String name, FinancialTransactionType type)
         this.creationDate = Instant.now();
     }
 
+    public void addUser(User user){
+        this.setUser(user);
+    }
+
+    public void removeUser(){
+        this.setUser(null);
+    }
 }
```

Comment:
These line 83-89 are necessary. Helper methods: add and remove should be on "to many" table. In this case in financialTransactionCategory

Reply:
Done, I removed redundant methods from User class.